filer/models/virtualitems.py

```diff
@@ -48,8 +48,20 @@ class UnsortedImages(DummyFolder):
     is_unsorted_uploads = True
     _icon = "unfiled_folder"
 
+    def __init__(self, user=None):
+        super().__init__()
+        self.user = user
+
     def _files(self):
-        return File.objects.filter(folder__isnull=True)
+        """
+        If the current user is not a superuser, we don't them to see/edit/change
+        file in the unsorted uploads folder that they don't own.
+        """
+        if self.user.is_superuser:
```

Comment:
For backwards compatibility, you could change this to
```suggestion
        if self.user.is_superuser or not filer_settings.FILER_ENABLE_PERMISSIONS:
```
This should still show all files if `FILER_ENABLE_PERMISSIONS` is `False`